src/TabViewPanResponder.js

```diff
@@ -20,6 +20,7 @@ function forHorizontal(props: Props) {
 
   let lastValue = null;
   let isMoving = null;
+  let deadzone = 20;
```

Comment:
nit: it's a constant, so can you declare it above the function as `const DEAD_ZONE = 20`? Sorry for nitpicking.